runtime/jcl/common/mgmtthread.c

```diff
@@ -1254,10 +1218,11 @@ initIDCache(JNIEnv *env)
 	if (NULL != gcls) {
 		(*env)->DeleteGlobalRef(env, gcls);
 	}
-	if (JNI_ENOMEM == err) {
+	// If we reach thus far without any exceptions
```

Comment:
`thus` -> `this`

Reply:
Also, `//` is not allowed in C files. Please reformat the comment (on a single line would be fine).

Reply:
fixed both, thanks.